hunts/src/StatusCell.js

```diff
@@ -7,8 +7,12 @@ import { updatePuzzle } from "./puzzlesSlice";
 export default function StatusCell({ row, value }) {
   const { id: huntId } = useSelector((state) => state.hunt);
   const dispatch = useDispatch();
-  if (["SOLVING", "STUCK", "EXTRACTION"].includes(value)) {
-    return (
+  var statuses_to_display = ["SOLVING", "STUCK", "EXTRACTION"];
```

Comment:
It seems the convention is to use `const` for arrays, even if the _contents_ of them are modified, as long as the array variable itself is not reassigned.

https://stackoverflow.com/questions/46553062/what-is-the-preferred-declaration-convention-for-objects-or-arrays-const-or-let

Reply:
Thanks -- didn't know that. :)